README.md

```diff
@@ -100,335 +154,353 @@ immunopepper make_bg \
---no-remove-se \
---no-validate-sg \
-```
-The output of this command line is in `spladder_out/spladder/genes_graph_conf3.ENCFF713JNY.pickle` and the count
-file `graph/spladder/genes_graph_conf3.merge_graphs.pickle` and `graph/spladder/genes_graph_conf3.merge_graphs.count.hdf5`.
-It takes around 7000 seconds for creating the merged splicegraph using a single thread.
-
-The mutation files *ImmunoPepper_usecase.maf* and *ImmunoPepper_usecase.vcf* are created arbitrarily. We
-just choose a position randomly and replace the original base with another one.
-
-### Get the mini-version
-Both the `ImmunoPepper_usecase.gtf` and `genome1.fa` are mini-version. The test cases only contain gene `ENSMUSG00000025902.13`
-and `ENSMUSG00000025903.14` and they are both in `chr1`. So we can just extract a small part
-of the annotation and genome sequence file.
+_________________________________________________________________________________________________________________________________________________________________
+kmer       |  coord                                  |  isCrossJunction   |  junctionAnnotated  |    readFrameAnnotated   |   sample 1  |     ...  |   sample n |
+_________________________________________________________________________________________________________________________________________________________________
+SDFQLIFVF  |    47970995:47971022:nan:nan:None:None  |      False         |        False        |           True          |       0.0   |     ...  |      0.0   |
 
+TLIPEELEP  |    47948807:47948834:nan:nan:None:None  |      False         |        False        |           False         |       0.0   |     ...  |      0.25  |
+_________________________________________________________________________________________________________________________________________________________________
 ```
-grep -E 'ENSMUSG00000025902.13|ENSMUSG00000025903.14' gencode.vM23.annotation.gtf > ImmunoPepper_usecase.gtf
-head -n 23252381 GRCm38.p6.genome.fa > genome1.fa
-```
-
-The SplAdder output contains more than 50000 genes. In our small usecase, we just contain 2 genes, whose id
-is 19 and 33. We can thus extract a mini-version splicegraph and its corresponding expression data, that is *ImmunoPepper_usecase.pickle* and *ImmunoPepper_usecase.count.hdf5*, to accelerate the running process.
-The corresponding script is `scripts/create_mini_mouse_data.py`.
-You can also use the original data *genes_graph_conf3.merge_graphs.pickle* and *genes_graph_conf3.merge_graphs.count.hdf5* to have a full run.
-
+6. **\[mut_mode\]_graph_kmer_JuncExpr**: Folder containing a file with the expression levels of kmers located across exon junctions. Format: [kmer, coord, isCrossJunction, junctionAnnotated, readFrameAnnotated]
+    - *kmer*: str. The kmer sequence
+    - *coord*: str. The genomic coordinates of the kmer. Format: start_e1:end_e1:start_e2:end_e2:start_e3:end_e3. If the kmer is only crossing one junction, start_e3 and end_e3 will be `None`.
```

Comment:
Almost. It actually contains sorted exon coordinates in the format:  left_e1:right_e1:left_e2:right_e2:left_e3:right_e3, with left < right. 
This corresponds to: 
For + strand: 
start_e1:end_e1:start_e2:end_e2:start_e3:end_e3
For - strand: 
end_e1:start_e1:end_e2:start_e2:end_e3:start_e3

Reply:
changed